### ISSUE TITLE
Return To Old Data Structure

### ISSUE THREAD

**Issue by CWrecker** [CSSEGISandData/COVID-19#1405]
I'll just make two notes here as a developer:

1. Before making any changes, please inform the community who is relying on your data. This could easily be a sample CSV file (In a test directory!) released a week (or at least 2 days) before its release. 

2. Please return to the old data structure, it was simply better.

Thanks :)

**Comment by anastasiiako** [comment]
Totally agree with CWrecker...

**Comment by shubhrampandey** [comment]
Yes I also agree. I have made a complete data visualization app based on the earlier data format and now....

**Comment by CWrecker** [comment]
Dear All, I have compiled the data and have reformated it into the original structure. Please let me know if you would like me to share it (Google Sheets)

**Comment by shubhrampandey** [comment]
Yes Please...I was trying to do that with the daily reports but incase you have done. Please share it.
Thanks a lot!!!!!

**Comment by shubhrampandey** [comment]
Link of my app: https://shubhrampandey.shinyapps.io/coronaVirusViz

**Comment by anastasiiako** [comment]
> Dear All, I have compiled the data and have reformated it into the original structure. Please let me know if you would like me to share it (Google Sheets)

We would highly appreciate it! <3 thank you in advance

**Comment by CWrecker** [comment]
Ok, let me just tidy it up, and then ill share it. 10 mins?

**Comment by shubhrampandey** [comment]
> Ok, let me just tidy it up, and then ill share it. 10 mins?

Thansks a lot!!!! Much appreciated..

**Comment by krismel109** [comment]
> Dear All, I have compiled the data and have reformated it into the original structure. Please let me know if you would like me to share it (Google Sheets)

Yes, please share!  Thank you!

**Comment by CWrecker** [comment]
---NOTICE---

Ok here we go:

Link to a live google sheets file:
https://docs.google.com/spreadsheets/u/1/d/e/2PACX-1vQisxSucfFNfhB_KJg1WmXZW_9_VtW_WxAy4ReE7vsRKDUgg2XwhOO0k1yS4V6tZnwZziRETZ3s1WBP/pubhtml?gid=0&single=true

Link to a live CSV file:
https://docs.google.com/spreadsheets/d/e/2PACX-1vQisxSucfFNfhB_KJg1WmXZW_9_VtW_WxAy4ReE7vsRKDUgg2XwhOO0k1yS4V6tZnwZziRETZ3s1WBP/pub?output=csv



Question for you, do you want the long and lat data as well?

**Comment by CWrecker** [comment]
Please let me know if the files work for your needs or you would like any other data (such as new cases each day).

**Comment by shubhrampandey** [comment]
Yes that is perfectly fine.....But hope you saw my app, that includes daily cases as well....

**Comment by CWrecker** [comment]
> Yes that is perfectly fine.....But hope you saw my app, that includes daily cases as well....

it won't let me connect. I keep getting: Dissconected from the server. let me try another device.

**Comment by CWrecker** [comment]
I have just added a JSON format above

**Comment by shubhrampandey** [comment]
I think you have scraped it from worldometer?

**Comment by CWrecker** [comment]
> I think you have scraped it from worldometer?

Yes I have. I forgot to mention that. It would appear they are much more reliable and faster to update. It might be a better source for the time being until JHU sorts out the data structures on their end.

**Comment by shubhrampandey** [comment]
Yes I was thinking the same. But I need time series cases as well....They have graph but data...Please let me know if you have any suggestion. I can read the graph via digitizer but that is the last option

**Comment by CWrecker** [comment]
> Yes I was thinking the same. But I need time series cases as well....They have graph but data...Please let me know if you have any suggestion. I can read the graph via digitizer but that is the last option

The same problem for me, i'll see what if I come up with. If JSU doesn't change the format of the time series too much, it might be possible to keep it.

**Comment by CWrecker** [comment]
> Yes that is perfectly fine.....But hope you saw my app, that includes daily cases as well....

Sorry, I keep getting the loading icon. :(

**Comment by shubhrampandey** [comment]
> > Yes that is perfectly fine.....But hope you saw my app, that includes daily cases as well....
> 
> Sorry, I keep getting the loading icon. :(

![image](https://user-images.githubusercontent.com/45200005/77431967-75e32880-6e03-11ea-97df-2139291e43b4.png)

Now you access...There was some error with backed...It takes approx 1 minute to load

**Comment by shubhrampandey** [comment]
Can I use JHU's daily reports files instead of time_series?

**Comment by CWrecker** [comment]
oh wow, this looks great

Ya, it might be able to compile daily reports into a time series

**Comment by shubhrampandey** [comment]
But JHU mentioned that recovered cases will not be updated in both daily and time series files. It is so disgusting man.

Can we rely on this: https://github.com/jbarton311/coronavirus-analysis/tree/master/coronavirus/output_data

**Comment by CWrecker** [comment]
I would recommend relying on the sources I posted above. The WorldOMeter seems to be the best source right now.

Ill try and put together a repository with more files.
